pwndbg/commands/version.py

```diff
@@ -93,13 +93,22 @@ def get_terminal_size():
     return f"Terminal width: {width_info}, height: {height_info}\n"
 
 
-@pwndbg.commands.ArgparsedCommand(
-    "Displays Pwndbg and its important deps versions.", category=CommandCategory.PWNDBG
-)
-def version() -> None:
+def version_impl() -> None:
+    """
+    Implementation of the `version` command.
+    """
     print("\n".join(map(message.system, all_versions())))
 
 
+if not pwndbg.dbg.name() == "lldb":
```

Comment:
```suggestion
if pwndbg.dbg.name() != "lldb":
```
why not just? any way lgtm I think

Reply:
That was a total brain fart wow